matcher.go

```diff
@@ -27,7 +27,23 @@ func (matcher *EqMatcher) FailureMessage() string {
 }
 
 func (matcher *EqMatcher) String() string {
-	return fmt.Sprintf("Eq(%v)", matcher.Value)
+	return fmt.Sprintf("Eq(%#v)", matcher.Value)
```

Comment:
I changed the format here because in the full message printed when a verification fails contains a list of other invocations and for those the parameters are formatted using `%#v` so was hard to compare the expected invocation to that list when matching more complex types like a map from string to some struct.